src/main/java/frc/robot/commands/IntakeCargo.java

```diff
@@ -0,0 +1,49 @@
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Intake;
+
+public class IntakeCargo extends CommandBase {
+  /** Creates a new IntakeCargo. */
+  private final Intake intake; 
+  public boolean isIntaking;
+
+  public IntakeCargo(Intake intake,boolean isIntaking) {
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(intake);
+    this.intake = intake;
+    this.isIntaking = isIntaking;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  
```

Comment:
```suggestion
```
Remove random empty line between the @override annotation and the function signature.